ci/cmake-format-all.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
is this script used anywhere? 🤔

Reply:
used by me to indent but we should integrate in to the CI

Reply:
Maybe we can add it to `pre-commit`?
https://github.com/cheshirekow/cmake-format-precommit

Reply:
I have added to the pre-commit just calling the script and then git diff --exit-code.

Simpler and we don't depend to another script that seems not really maintained